pmcelhaney/counterfact
# add some tests

- remove dead code
- test OperationTypeCoder with OpenAPI 2
- the requirement being undefined should not happen
- remove debugging aid
- remove a condition that "can't" happen
- requirment.toJSON() was not used anywhere
- test for schemaCoder.modulePath()
